doc/sphinx/proof-engine/ltac.rst

```diff
@@ -253,6 +260,21 @@ that won't clash with term names.  You can use :n:`ltac:(@name)` or :n:`(@name)`
 to control whether a :token:`name` is interpreted as, respectively, an |Ltac|
 variable or a term.
 
+Note that values from toplevel symbols, unlike locally-defined symbols, are
+substituted only when they appear at the beginning of an :token:`ltac_expr`:
```

Comment:
```suggestion
substituted only when they appear at the beginning of an :token:`ltac_expr` or :token:`tactic_arg`:
```
I'm a little bit confused here.  It is also the case that global names are substituted when given as arguments to Ltac functions, as in
```coq
Ltac n := match goal with _ => idtac "n call" end.
Ltac foo _ := idtac "foo call".
Ltac foo_n := foo n.

Goal True.
  foo_n.
(* n call
foo call *)
```
I don't entirely see how to match this up with the grammar in a way that's precise and accurate, but I think the suggestion I make in this comment is enough.

Reply:
Good point.  `idtac n` is an `l1_tactic` in the grammar, not a `qualid {+ tac_arg }`.  So it makes some sense.  Even if it's not precise I think the example alerts readers that things may not be totally consistent.

Reply:
> Linter complains about trailing whitespace.

That's on one of the earlier commits.  It should go away when I squash.

> Btw, there's still some text using `@tactic`, which I don't see defined anywhere:

I think I'll fix that later, it shouldn't cause much confusion.  There are a ton in the rsts that haven't been updated.